### PR TITLE
Due dates & type-o's

### LINKED ISSUE #3: Determine due dates
One of the remaining items is to determine the due dates for the files to be submitted and verified. I'd like to propose 10 days after the beginning of the month. I hope to avoid the sometimes overlap with the TNP reporting due dates and the earlier dates can be justified by the relatively simpler reporting requirements.

If that was the case, this would be the due dates:

  * June 11th
  * July 10th
  * August 10th
  * September 10th
  * October 10th
 
For October/November, since the reporting period is inclusive of November 1st, we can push one more day so that reporting deadline is November 12th (due to the weekend).

Thoughts, @nicklucius @levyj?

### PR DESCRIPTION
**Pull request by tomschenkjr**
This pull request both resolves #3 and #4. I've put it under a more generic `dev` branch for now.

### CODE REVIEW

File: _docs/trips.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ order: 1
 
 ## Scope
 
-All Trips completed during the reporting period.  Trips in progress at the end of the reporting period should be held for the next report.  For the purposes of this file, a trip is a transaction with a specific customer. Trips that end [**sentence fragment; not sure what the end was going to be**]
+All Trips completed during the reporting period.  Trips in progress at the end of the reporting period should be held for the next report.  For the purposes of this file, a trip is a transaction with an single rider.
```
Comment (nicklucius) [inline _docs/trips.md]: Change "an" to "a" before single rider.

**Review by nicklucius — COMMENTED** [review]
It looks like this will have a merge conflict. I think the dev branch was based off of gh-pages prior to #5.

**Comment by tomschenkjr** [comment]
@nicklucius - thanks. I've rebased on the new `gh-pages` and also fixed the type-o. Please re-review.

**Review by nicklucius — APPROVED** [review]
Looks good

**Pull request merged**